src/pill/pill.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { PillProps } from "./types";
+import { Content, Icon, Label, Wrapper } from "./pill.style";
+
+export const Pill = ({
+    type,
+    colorType = "black",
+    className,
+    children,
+    icon,
+    ...otherProps
+}: PillProps): JSX.Element => {
+    return (
+        <Wrapper
```

Comment:
spread the remaining props to `Wrapper` since the component inherits from a div element